### PR TITLE
switch organization - backend

### PR DESCRIPTION
**Pull request by ChyzhykTech**
Implemented switch organization functionality (backend only).
I have implemented some methods from userService.ts and created orgatizationRepository.ts.
Think that these changes are important for the team.

### CODE REVIEW

File: backend/src/api/routes/userRoute.ts [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+import { User } from '../../data/entities/User';
 
 const router = Router();
 
 router
   .get('/', run(getUsers))
   .post('/me', run((req: Request) => getUserDataFromToken(req.user as ITokenData)))
-  .get('/:id', run((req: Request) => getUserById(req.params.id)));
+  .get('/:id', run((req: Request) => getUserById(req.params.id)))
+  // eslint-disable-next-line max-len
```
Comment (AlikRakhmonov) [inline backend/src/api/routes/userRoute.ts]: do not ignore errors!

File: backend/src/api/routes/userRoute.ts [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
+  .put('/switch-orgatization/:id', run((req: Request) => switchUserOrganization(req.user as User, req.params.id)));
```
Comment (AlikRakhmonov) [inline backend/src/api/routes/userRoute.ts]:
```suggestion
  .put('/:id/switch-orgatization', run((req: Request) => switchUserOrganization(req.user as User, req.params.id)));
```

File: backend/src/services/userService.ts [hunk 3]
```diff
@@ -1,11 +1,30 @@
...
+  const user = await userRepository.getById(id);
+  return extractTransportedUser(user);
+};
+
+export const switchUserOrganization = async (user: User, id: string) => {
+  const userRepository = getCustomRepository(UserRepository);
+  const organizationRepository = getCustomRepository(OrganizationRepository);
+
+  const nextUserOrgatization = await organizationRepository.getById(id);
+  // eslint-disable-next-line no-param-reassign
```
Comment (AlikRakhmonov) [inline backend/src/services/userService.ts]: remove it

**Review by AlikRakhmonov — CHANGES_REQUESTED** [review]

File: backend/src/services/userService.ts (hunk 3 above)
Comment (ChyzhykTech) [reply backend/src/services/userService.ts]: solved

**Review by AlikRakhmonov — APPROVED** [review]

**Pull request merged**